File: C#/3.cs
using System;

class Program
{
    static int CountEven(long num)
    {
        int counter = 0;
        while (num != 0)
        {
            long n = num % 10;
            if (n % 2 == 0 && n != 0) counter++;
            num /= 10;
        }
        return counter;
    }

    static void Main()
    {
        int n = int.Parse(Console.ReadLine());

        for (int i = 0; i < n; i++)
        {
            long tmp = long.Parse(Console.ReadLine());
            Console.WriteLine(CountEven(tmp));
        }
    }
}

File: Swift/3.swift
func countEven(_ num: Int) -> Int {
    var counter = 0
    var num = num
    while num != 0 {
        let n = num % 10
        if (n % 2 == 0) && (n != 0) { counter += 1 }
        num /= 10
    }
    return counter
}

if let n = Int(readLine() ?? "0") {
    for _ in 0..<n {
        if let tmp = Int(readLine() ?? "0") {
            print(countEven(tmp))
        }
    }
}
